package.json

```diff
@@ -39,14 +39,16 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run test & npm run build"
+      "pre-push": "node tools/deps-duplication-detector.js && (npm run test & npm run build)"
```

Comment:
We maybe should be the dup checker into the scripts an just have `npm run check` or so here.